### PR TITLE
[high] Potential Checkout Bug in iOS SDK

### PR DESCRIPTION
**Pull request by hoyelo**

### CODE REVIEW

File: AffirmSDK/AffirmCheckoutViewController.m [hunk 1]
```diff
@@ -189,10 +189,12 @@ - (void)webView:(WKWebView *)webView decidePolicyForNavigationAction:(WKNavigati
     NSURL *URL = navigationAction.request.URL;
     NSString *strippedURL = [NSString stringWithFormat:@"%@://%@%@", URL.scheme, URL.host, URL.path];
     if ([strippedURL isEqualToString:AFFIRM_CHECKOUT_CONFIRMATION_URL]) {
+        BOOL isResponded = NO;
```
Comment (kleinlieu) [inline AffirmSDK/AffirmCheckoutViewController.m]: Can we have a better name for this, maybe `hasRespondedWithCheckoutToken`?

File: AffirmSDK/AffirmRequest.m [hunk 2]
```diff
@@ -181,7 +181,7 @@ - (NSDictionary *)parameters
         @"public_api_key": self.publicKey,
         @"user_confirmation_url": AFFIRM_CHECKOUT_CONFIRMATION_URL,
         @"user_cancel_url": AFFIRM_CHECKOUT_CANCELLATION_URL,
-        @"user_confirmation_url_action": @"POST",
+        @"user_confirmation_url_action": @"GET",
```
Comment (kleinlieu) [inline AffirmSDK/AffirmRequest.m:184]: What is the intention behind this change? Could this be parameterized from the call site instead?

**Review by kleinlieu — COMMENTED** [review]
Some general comments left behind. I would recommend testing this first before issuing another hotfix release into the wild.

File: AffirmSDK/AffirmCheckoutViewController.m (hunk 1 above)
Comment (hoyelo) [reply AffirmSDK/AffirmCheckoutViewController.m]: Okay.

File: AffirmSDK/AffirmRequest.m (hunk 2 above)
Comment (hoyelo) [reply AffirmSDK/AffirmRequest.m:184]: It's required by checkout api, and that's why it failed to get checkout token.

**Review by vcello — APPROVED** [review]

**Pull request merged**